docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -133,3 +133,125 @@ sequenceDiagram
     Note over autopilot: End
 ```
 
+## Run locally end to end
+
+In this tutorial we will run all the components locally on Ethereum mainnet, including the orderbook, autopilot, driver and solver engine. All these components have their own configuration parameters. For the sake of simplicity, we will use the bare minimum configuration for all of them.
+
+## Solver engine configuration file
+
+baseline.toml
+```
+chain-id = "1" # Ethereum mainnet
+base-tokens = [
+    "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
```

Comment:
Would be nice to add the name of the tokens as a comment.

Reply:
Added.